Does not make in Kali Linux (Debian), unable to locate `libprocps.ps` and `proc/readproc.h` not found.

Attempted two methods of compilation: 1) via standard execution of `make`, 2) via use of docker to build. Both methods failed with `src/xutil.cpp:17:10: fatal error: proc/readproc.h: No such file or directory`. The first method (standard make) also was unable to locate `libprocps.pc`, which does not appear to be included in the `libprocps7` package.

--- a/README.md
+++ b/README.md
@@ -183,14 +183,26 @@ Prepare build directory
 
 ```bash
 mkdir -p build
+```
+
+Use one of:
+
+Ubuntu
+```sh
 docker run --rm -it -v $(pwd)/build:/src --name ubuntu ubuntu:20.04 bash
+apt-get update
+apt-get install -y build-essential libxrandr-dev libxcursor-dev git-core pkg-config libprocps-dev libgtest-dev libgmock-dev
+```
+
+OR Arch
+```sh
+docker run --rm -it -v $(pwd)/build:/src --name archlinux archlinux:base bash
+pacman -Sy git gcc make pkgconfig libxcursor xorg-xrandr gtest gmock
 ```
 
 Being inside the container run
 
-```bash
-apt-get update
-apt-get install -y build-essential libxrandr-dev libxcursor-dev git-core pkg-config libprocps-dev libgtest-dev libgmock-dev
+```sh
 cd /src
 git clone --depth 1 https://github.com/alex-courtis/xlayoutdisplay .
 make

--- a/README.md
+++ b/README.md
@@ -189,7 +189,7 @@ Use one of:
 
 Ubuntu
 ```sh
-docker run --rm -it -v $(pwd)/build:/src --name ubuntu ubuntu:20.04 bash
+docker run --rm -it -v $(pwd)/build:/src --name ubuntu ubuntu:22.04 bash
 apt-get update
 apt-get install -y build-essential libxrandr-dev libxcursor-dev git-core pkg-config libprocps-dev libgtest-dev libgmock-dev
 ```

--- a/README.md
+++ b/README.md
@@ -190,7 +190,7 @@ Being inside the container run
 
 ```bash
 apt-get update
-apt-get install -y build-essential libxrandr-dev libxcursor-dev git-core
+apt-get install -y build-essential libxrandr-dev libxcursor-dev git-core pkg-config libprocps-dev libgtest-dev libgmock-dev
 cd /src
 git clone --depth 1 https://github.com/alex-courtis/xlayoutdisplay .
 make

